pkg/kubectl/cmd/util/helpers.go

```diff
@@ -62,16 +63,40 @@ func AddSourceToErr(verb string, source string, err error) error {
 	return err
 }
 
+var fatalErrHandler = fatal
```

Comment:
I'm fine with this global variable.  It behaves with an intent similar to `OPENSHIFT_ON_PANIC`.  I'm also ok with the alternative of checking an env var each time down the error path.  I don't want to expose the handler on the public `CheckErr`.

I guess I could see the advantage using an env var to avoid ever having it change during an execution, but I don't feel strongly.